OrbitLinuxTracing/TracerThread.h

```diff
@@ -156,6 +157,7 @@ class TracerThread {
   ContextSwitchManager context_switch_manager_;
   std::unique_ptr<PerfEventProcessor> uprobes_event_processor_;
   std::unique_ptr<GpuTracepointEventProcessor> gpu_event_processor_;
+  absl::flat_hash_map<uint64_t, orbit_grpc_protos::TracepointInfo> stream_id_to_tracepoint_;
```

Comment:
Move this up after `callchain_sampling_ids_` so that all members related to stream ids are together. Also I think this should be called something like `ids_to_tracepoint_info_`: the other members don't mention "stream", and let's mention the full name of `TracepointInfo` in this name.